docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -151,7 +151,7 @@ Teleport will respond back with a discovery info:
 ```bash
 $ curl https://tele.example.com/webapi/ping
 
-{"server_version":"5.0.0-dev","min_client_version":"3.0.0"}
+{"auth":{"type":"local","second_factor":"otp"},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3026"},"ssh":{"listen_addr":"[::]:3023","tunnel_listen_addr":"0.0.0.0:3024","public_addr":"tele.example.com:443"},"db":{}},"server_version":"6.0.1","min_client_version":"3.0.0"}
```

Comment:
I removed those on purpose to focus attention on the key details

Reply:
Updated to the shorter version but updated the `server_version` to 6.0.0